datasource/http/test-fixtures/invalid_method.pkr.hcl

```diff
@@ -0,0 +1,26 @@
+source "null" "example" {
+  communicator = "none"
+}
+
+data "http" "basic" {
+  url = "https://www.packer.io/"
+  method = "DELETE"
+}
+
+locals {
+  url = "${data.http.basic.url}"
+  body = "${data.http.basic.body}" != ""
+}
+
+build {
+  name = "mybuild"
+  sources = [
+    "source.null.example"
+  ]
+  provisioner "shell-local" {
+    inline = [
+      "echo url is ${local.url}",
+      "echo body is ${local.body}"
```

Comment:
I would suggest removing those as the test may not work on Windows otherwise.

Note: they do succeed now as the acceptance tests are not run for every PR, but once we run them, this will probably fail.

Note 2: though come to think of it, `echo` is a valid command on powershell, since it's interpolated before execution, this might work in the end?

Reply:
This was copied form an existing test `basic.pkr.hcl`. Should this still be changed?